lib/elixir/src/elixir_erl.erl

```diff
@@ -318,7 +318,12 @@ typespecs_form(Map, TranslatedTypespecs, MacroNames) ->
   Forms1 = types_form(Types, Forms0),
   Forms2 = callspecs_form(spec, Specs, [], MacroNames, Forms1, Map),
   Forms3 = callspecs_form(callback, AllCallbacks, OptionalCallbacks, MacroCallbackNames, Forms2, Map),
-  {Types, AllCallbacks, Forms3}.
+
+  AllCallbacksWithoutSpecs = lists:usort([
+    {Kind, NameArity} || {Kind, NameArity, _Line, _Spec} <- AllCallbacks
```

Comment:
I was thinking we could already return `{Kind, Name, Arity}` here. So in the line 524, we would do `Callback <- Callbacks`.

Reply:
Right. I saw that in other place we had `{Kind, {Name, Arity}, _, _}` (specs I believe) but this sounds good too, I'll make the change.

Reply:
ok, now I see why it's nicer :)